rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.2.2-release-notes.adoc

```diff
@@ -0,0 +1,52 @@
+= RPA Builder 6.2.2 Release Notes
+
+RPA Builder is the on-premise application that enables you to build automations by specifying the action steps of the process.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== May 5, 2023
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* Updated the EPPlus .NET library to version 6.2.0.
+* Added property in Excel Controlled Loop and Read from Excel file action steps to avoid recalculation of formulas in date cells.
```

Comment:
```suggestion
* Added property in *Excel Controlled Loop* and *Read from Excel file* action steps to avoid recalculation of formulas in date cells.
```